File: Unity/QuestOfTheRoundTable/Assets/Scripts/Objects/FoeCard.cs
namespace Card
{
    public class FoeCard : AdventureCard
    {
        public FoeCard(string name) : base(name)
        {
        }
        public FoeCard(string name, int bp) : base(name)
        {
            BattlePoints = bp;
        }
        public FoeCard(string name, int bp, int nbp) : base(name)
        {
            BattlePoints = bp;
            NamedBattlePoints = nbp;
        }
    }
}

File: JavaFXQTRT/src/main/java/src/game_logic/FoeCard.java
package src.game_logic;

public class FoeCard extends AdventureCard{
	
	public FoeCard(String name, int battlePoints, TYPE type) {
		super(name, battlePoints, type);
	}
	
	public FoeCard(String name, int battlePoints, int namedBattlePoints, TYPE type) {
		super(name, battlePoints, namedBattlePoints, type);
	}

	public FoeCard(String card, String path) {
		super(card,path);
		type = TYPE.FOES;
	}
}